src/core/dom/modal.js

```diff
@@ -133,10 +135,10 @@ function getPotentiallyFocusableElements(element) {
  *
  * @param {HTMLElement} element The Element top operate on.
  * @param {string} attribute  The name of the attribute.
- * @param {?string} value The value of the attribute.
+ * @param {?string|undefined} value The value of the attribute.
```

Comment:
If this gets passed undefined intentionally, ignore. If the arg is optional, see below

```suggestion
 * @param {?string=} value The value of the attribute.
```

Reply:
This worked! Updating